sagemaker-python-sdk/tensorflow_script_mode_training_and_serving/tensorflow_script_mode_training_and_serving.ipynb

```diff
@@ -0,0 +1,253 @@
+    "                             framework_version='1.12',\n",
+    "                             py_version = 'py3',\n",
+    "                             distributions = {'parameter_server': {'enabled': True}})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Calling ``fit``\n",
+    "\n",
+    "To start a training job, we call `estimator.fit(training_data_uri)`.\n",
+    "\n",
+    "You can pass an ``s3_input``, a dictionary or an S3 location to the ``fit`` method. If a dictionary is used the keys, named **channels**, \n",
+    "have values pointing to the data location. `estimator.fit(training_data_uri)` downloads the TensorFlow container with TensorFlow Python 3, GPU version to the SageMaker training job instances. If a S3 location is used the estimator will create a default channel named ``'training'`` which points to this S3 location.\n",
+    "\n",
+    "When training starts, the TensorFlow container executes mnist.py, passing `hyperparameters` which we don't have any in our case and `model_dir` as script arguments, executing the example as follows:\n",
+    "```bash\n",
+    "python mnist.py --model_dir s3://<DEFAULT_BUCKET>/<TRAINING_JOB>\n",
+    "```\n",
+    "In the end of the training, the training job will upload the saved model for TensorFlow serving."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mnist_estimator.fit(training_data_uri)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# Deploy the trained model to an endpoint\n",
+    "\n",
+    "The ``deploy()`` method creates an SageMaker model which is then deployed to an endpoint to serve prediction requests in real time. We will use the TensorFlow Serving container for the endpoint because we trained with script mode. This serving container runs an implementation of a web server which is compatible with SageMaker hosting protocol. The [Using your own inference code]() document explains how inference images are run in SageMaker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictor = mnist_estimator.deploy(initial_instance_count=1, instance_type='ml.p2.xlarge')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Invoke the endpoint\n",
+    "\n",
+    "Let's download the training data and use that as input for inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "!aws s3 cp s3://sagemaker-sample-data-{region}/tensorflow/mnist/train_data.npy train_data.npy\n",
+    "!aws s3 cp s3://sagemaker-sample-data-{region}/tensorflow/mnist/train_labels.npy train_labels.npy\n",
```

Comment:
same comment here about region compatibility. Previous notebooks switched to https (#422, #423), but I think adding the `--region` flag should work as well